pkg/registry/core/service/storage/rest_test.go

```diff
@@ -2150,9 +2156,10 @@ func TestServiceRegistryExternalTrafficHealthCheckNodePortUserAllocation(t *test
 	svc := &api.Service{
 		ObjectMeta: metav1.ObjectMeta{Name: "external-lb-esipp"},
 		Spec: api.ServiceSpec{
-			Selector:        map[string]string{"bar": "baz"},
-			SessionAffinity: api.ServiceAffinityNone,
-			Type:            api.ServiceTypeLoadBalancer,
+			Selector:                      map[string]string{"bar": "baz"},
+			SessionAffinity:               api.ServiceAffinityNone,
+			Type:                          api.ServiceTypeLoadBalancer,
+			AllocateLoadBalancerNodePorts: utilpointer.BoolPtr(true),
```

Comment:
Sadly a change just went it which will force you to rebase this (this file in particular).  You can add AllocateLoadBalancerNodePorts to the pkg/api/service/testing/make.go `MakeService` and get all of these for free.